pyopenephys/openephys_tools.py

```diff
@@ -17,7 +17,7 @@
 MAX_NUMBER_OF_RECORDS = int(1e6)
 MAX_NUMBER_OF_EVENTS = int(1e6)
 
-def loadContinuous(filepath, dtype=float):
+def loadContinuous(filepath, dtype=np.int16):
```

Comment:
I think it's safer to have these as strings
```suggestion
def loadContinuous(filepath, dtype='int16'):  
```